operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Domain.java

```diff
@@ -693,6 +702,18 @@ private void addIllegalSitConfigForMii() {
       }
     }
 
+    private void verifyIstioExposingDefaultChannel() {
+      if (spec.isIstioEnabled()) {
+        spec.getAdminServer().getAdminService().getChannels().forEach(this::checkForDefaultNameExposed);
+      }
+    }
+
+    private void checkForDefaultNameExposed(Channel channel) {
+      if (channel.getChannelName().contains("-default")) {
```

Comment:
Shouldn't this be checking for "default", "default-secure" or "default-admin"?

Reply:
The introspectDomain.py already splitted the channel and passed them to the operator, so the channels are xxx-default filtered in 

```
private void checkForDefaultNameExposed(Channel channel) {
      if (channel.getChannelName().contains("-default")) {
        failures.add(DomainValidationMessages.cannotExposeDefaultChannelIstio());
      }
    }```

Should we check fo others like *-snmp, *-iiop *-ldap

Reply:
fixed